### PR TITLE
Modify Index mapping curl to include necessary Content-Type Header fo…

### PR DESCRIPTION
**Pull request by shakalandy**
- fix Index mapping curl to include necessary Content-Type Header for PUT requests.
- include link to ES documentation on index mapping
- modify some other links

#593  can be closed due to commit user mess and layer 8 problems, thanks ;)

### CODE REVIEW

File: pages/configuration/elasticsearch.rst [hunk 1]
```diff
@@ -94,7 +94,7 @@ system defaults allow. **Set it to at least 64000 open file descriptors.**
 
 Graylog will show a notification in the web interface when there is a node in the Elasticsearch cluster which has a too low open file limit.
 
-Read about how to raise the open file limit in the corresponding `2.x <https://www.elastic.co/guide/en/elasticsearch/reference/2.3/setup-configuration.html#file-descriptors>`__ / `5.x <https://www.elastic.co/guide/en/elasticsearch/reference/5.6/file-descriptors.html>`__ documentation pages.
+Read about how to raise the open file limit in the corresponding `2.x <https://www.elastic.co/guide/en/elasticsearch/reference/6.6/setup-configuration.html#file-descriptors>`__ / `6.x <https://www.elastic.co/guide/en/elasticsearch/reference/6.6/file-descriptors.html>`__ documentation pages.
```
Comment (jalogisch) [inline pages/configuration/elasticsearch.rst]: you should fix this links correctly - as ES5 and ES6 are supported and your half fixed link writes ES2 but links to ES6 ...

**Review by jalogisch — CHANGES_REQUESTED** [review]
The one little piece that neeed to be corrected. But all other is fine.

thank you for this fix!

File: pages/configuration/elasticsearch.rst (hunk 1 above)
Comment (shakalandy) [reply pages/configuration/elasticsearch.rst]: changed to wrong link to 2.3, all other to 6.7

File: pages/configuration/elasticsearch.rst [hunk 2]
```diff
@@ -94,7 +94,7 @@ system defaults allow. **Set it to at least 64000 open file descriptors.**
 
 Graylog will show a notification in the web interface when there is a node in the Elasticsearch cluster which has a too low open file limit.
 
-Read about how to raise the open file limit in the corresponding `2.x <https://www.elastic.co/guide/en/elasticsearch/reference/6.6/setup-configuration.html#file-descriptors>`__ / `6.x <https://www.elastic.co/guide/en/elasticsearch/reference/6.6/file-descriptors.html>`__ documentation pages.
+Read about how to raise the open file limit in the corresponding `2.x <https://www.elastic.co/guide/en/elasticsearch/reference/2.3/setup-configuration.html#file-descriptors>`__ / `6.x <https://www.elastic.co/guide/en/elasticsearch/reference/6.7/file-descriptors.html>`__ documentation pages.
```
Comment (jalogisch) [inline pages/configuration/elasticsearch.rst]: theES 2.x link does not make sense - it must be 5.x for the name and the link needs to point to 5.6 what is the latest stable.

**Review by jalogisch — CHANGES_REQUESTED** [review]
please adjust the link

File: pages/configuration/elasticsearch.rst (hunk 2 above)
Comment (shakalandy) [reply pages/configuration/elasticsearch.rst]: @jalogisch there are more 2.x links in the Doc, should they all be moved to 6.x then? or 5.6?

Comment (jalogisch) [reply pages/configuration/elasticsearch.rst]: only 5.x and 6.x are supported. No need to link to unsupported versions documentation.

Comment (jalogisch) [reply pages/configuration/elasticsearch.rst]: To answer the question: I personal would move to 6.x if it makes sense.

**Review by jalogisch — APPROVED** [review]

**Pull request merged**